### ISSUE TITLE
Masquerading eventually fails

### ISSUE THREAD

**Issue by lstoll** [lstoll/k8s-vpcnet#32]
After running a bunch of pods on an instance, traffic to the internet seems to fail. A cursory look shows that no iptables entry was added, but unclear why.

**Comment by lstoll** [comment]
Seems to be a weird consistency thing? e.g

```
I0710 16:24:01.348620   15869 ipmasq_linux.go:72] Creating chain "CNI-4a49c9e7f0c42b71df4c7b9e" for podID "260cbcbdd1fcede3cecf19abd83c2ace241e53885dfa1a71a44a275176c70e19"
I0710 16:24:01.349439   15869 ipmasq_linux.go:80] Adding skip rule 10.0.0.0/18 for podID "260cbcbdd1fcede3cecf19abd83c2ace241e53885dfa1a71a44a275176c70e19"
I0710 16:24:01.351089   15869 ipmasq_linux.go:80] Adding skip rule 100.64.0.0/10 for podID "260cbcbdd1fcede3cecf19abd83c2ace241e53885dfa1a71a44a275176c70e19"
    /Users/lincoln.stoll/src/github.com/lstoll/k8s-vpcnet/pkg/cni/ipmasq/ipmasq_linux.go:82
E0710 16:24:01.353066   17381 cni.go:294] Error adding network: Error inserting IPTables rule for Container ID "260cbcbdd1fcede3cecf19abd83c2ace241e53885dfa1a71a44a275176c70e19": Error appending skip rule for 100.64.0.0/10 to (pre-existing? false) chain "CNI-4a49c9e7f0c42b71df4c7b9e": exit status 1: iptables: No chain/target/match by that name.
```

seems to clearly show it being created, and one rule also being created first.

**Comment by lstoll** [comment]
Fixed in #35
